view/frontend/web/js/view/payment/method-renderer/rvvup-method.js

```diff
@@ -502,6 +502,13 @@ define([
             },
             getPayLaterConfigValue: function (key) {
                 let values = rvvup_parameters
+
+                const paypalLoaded = !!values?.settings?.paypal?.checkout;
+
+                if (!paypalLoaded) {
```

Comment:
This throwing the undefined error because it is specifically called by this template & method https://github.com/rvvup/magento-plugin/blob/72a3d05e39f0144e30266e5741d2d7c95970feaf/view/frontend/web/template/payment/rvvup.html#L43

This template is used across all payment methods and sets several placeholders & effectively, this is a simple placeholder to decide whether to display or not the paylater messaging.
In our case, if checkout js object does not exist, it means that the PayPal payment method is not loaded at all. As such, the false return value would not matter.

I think the main issue is that we cannot dymanically enable/disable the whole component, that's why we used the display/hide style setting in the first place. Or it might need to be reworked with knockout if statements to separate concerns even more.